.github/workflows/golangci-lint.yml

```diff
@@ -0,0 +1,24 @@
+name: Lint Go Code
+
+on:
+  pull_request:
+    paths:
+      - '**/*.go'
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
```

Comment:
There is already v4 for this action:
https://github.com/actions/checkout

I suggest using the latest version